### DISCUSSION TITLE
chia is not cyncing

### DISCUSSION THREAD

**Discussion by Jangir92** [Chia-Network/chia-blockchain#9293] [Support]
hello. my chia is not syncing. I waited 7 days. then I deleted the db folder. Synchronization started from the very beginning and reached 552 064. I have already tried to delete the db folder and the chia program 3 times. but all the same is synchronized only before this 552 064.please help me with my problem

**Comment by AleksandrPDA** [comment]
Привет Это Александр, что то поддержка тут мне тоже пока ничего не подсказала, будем ждать, я уже устал, что только не пробовал.

**Comment by AleksandrPDA** [comment]
This issue has never been resolved. Why does the support service not help in solving our problem?
